lib/process-services-cloud/src/lib/process-cloud/services/process-cloud.service.ts

```diff
@@ -0,0 +1,133 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AlfrescoApiService, FormValues, AppConfigService, LogService } from '@alfresco/adf-core';
+import { Injectable } from '@angular/core';
+import { Observable, from, throwError } from 'rxjs';
+import { map, catchError } from 'rxjs/operators';
+import { ProcessDefinitionRepresentationCloud } from '../models/process-definition-cloud.model';
+import { ProcessInstanceVariableCloud } from '../models/process-instance-variable-cloud.model';
+import { ProcessInstanceCloud } from '../models/process-instance-cloud.model';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ProcessCloudService {
+
+    constructor(private alfrescoApiService: AlfrescoApiService,
+                private appConfigService: AppConfigService,
+                private logService: LogService) {
+    }
+
+    /**
+     * Gets process definitions associated with an app.
+     * @param appId ID of a target app
+     * @returns Array of process definitions
+     */
+    getProcessDefinitions(appName?: string): Observable<ProcessDefinitionRepresentationCloud[]> {
+
+        if (appName) {
+            let queryUrl = this.buildQueryUrl(appName);
```

Comment:
doesn't make sense hide the API url in a method 
we should save the host in a class variable and do 
`${this.host}/${appName}-rb/v1/process-definitions`